### PR TITLE
# JDP191201-27

### PR DESCRIPTION
**Pull request by DariuszKny**
-GroupController (added endpoints)
-Add Group and GroupDto Class

### CODE REVIEW

File: src/main/java/com/kodilla/ecommercee/GroupController.java [hunk 1]
```diff
@@ -1,4 +1,35 @@
...
+@RequestMapping("v1/group")
 public class GroupController {
+
+    @GetMapping(value = "getGroups")
+
+    public List<GroupDto> getGroups() {
+        return new ArrayList<>();
+    }
+    @GetMapping(value = "getGroup")
+    public GroupDto getGroup(@RequestParam long groupId){
```
Comment (PawelM-code) [inline src/main/java/com/kodilla/ecommercee/GroupController.java]: long na Long

File: src/main/java/com/kodilla/ecommercee/GroupController.java [hunk 2]
```diff
@@ -1,4 +1,35 @@
...
+        return new GroupDto(groupId,"Some Group Name");
+    }
+
+    @DeleteMapping(value = "deleteGroup")
+    public void deleteGroup(@RequestParam long groupId) {
```
Comment (PawelM-code) [inline src/main/java/com/kodilla/ecommercee/GroupController.java]: j.w long na Long

File: src/main/java/com/kodilla/ecommercee/GroupController.java [hunk 3]
```diff
@@ -1,4 +1,35 @@
 package com.kodilla.ecommercee;
 
+import com.kodilla.ecommercee.domain.GroupDto;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("v1/group")
```
Comment (radoslaw-lazur) [inline src/main/java/com/kodilla/ecommercee/GroupController.java]: "/v1/group

File: src/main/java/com/kodilla/ecommercee/GroupController.java (hunk 1 above)
Comment (DariuszKny) [reply src/main/java/com/kodilla/ecommercee/GroupController.java]: poprawiono

File: src/main/java/com/kodilla/ecommercee/GroupController.java (hunk 2 above)
Comment (DariuszKny) [reply src/main/java/com/kodilla/ecommercee/GroupController.java]: j.w

File: src/main/java/com/kodilla/ecommercee/GroupController.java (hunk 3 above)
Comment (DariuszKny) [reply src/main/java/com/kodilla/ecommercee/GroupController.java]: j.w

**Review by PawelM-code — APPROVED** [review]

**Review by MichalOrlowski — APPROVED** [review]

**Review by interceptor2048 — APPROVED** [review]

**Pull request merged**